packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1327,13 +1316,6 @@ Posts.addView("personalTagProgressPosts", (terms: PostsViewTerms) => {
   }
 })
 
-ensureIndex(Posts,
-  augmentForDefaultView({ userId: 1, tagRelevance: 1}),
-  {
-    name: "posts.users_tagged_posts",
```

Comment:
This was originally made to support the Tagging Dashboard project, which included a page where users could see a list of their posts that hadn't yet been tagged. 

If we think we're likely to ever bring this back, I could comment it out rather than delete it.